web/src/components/organisms/Settings/Members/index.tsx

```diff
@@ -126,7 +133,11 @@ const Members: React.FC = () => {
         <ActionHeader>
           <Search placeholder={t("search for a member")} allowClear style={{ width: 264 }} />
         </ActionHeader>
-        <Table dataSource={dataSource} columns={columns} style={{ padding: "24px" }} />
+        <Table
+          dataSource={dataSource}
+          columns={columns}
+          style={{ padding: "24px", overflow: "auto" }}
```

Comment:
For the table we just want scroll on the horizontal pane, right? So please use overflowX

Reply:
Fixed.